yasnippet.el

```diff
@@ -3286,58 +3300,29 @@ If so cleans up the whole snippet up."
 
 ;; Apropos markers-to-points:
 ;;
-;; This was found useful for performance reasons, so that an
-;; excessive number of live markers aren't kept around in the
-;; `buffer-undo-list'. However, in `markers-to-points', the
+;; This was found useful for performance reasons, so that an excessive
+;; number of live markers aren't kept around in the
+;; `buffer-undo-list'.  However, in `markers-to-points', the
 ;; set-to-nil markers can't simply be discarded and replaced with
-;; fresh ones in `points-to-markers'. The original marker that was
-;; just set to nil has to be reused.
+;; fresh ones in `points-to-markers'.  The original marker that was
+;; just set to nil has to be reused (why?).
```

Comment:
> The experiment mentioned above disregards the "performance reasons" mentioned in the comment block, but I don't recall actually measuring any slowdown

Ah, it might refer to this text in `(elisp) Overview of Markers` then:
```
   Insertion and deletion in a buffer must check all the markers and
relocate them if necessary.  This slows processing in a buffer with a
large number of markers.  For this reason, it is a good idea to make a
marker point nowhere if you are sure you don't need it any more.
Markers that can no longer be accessed are eventually removed (*note
Garbage Collection::).
```
> I just quickly experimented with simply removing the calls to `yas--markers-to-points` and `yas--points-to-markers` and everything seems to be working.

I guess skipping conversion of markers to positions could cause problems after expanding 100s or 1000s of snippets. You wouldn't notice with a quick test.

Anyway, I'm not sure it's worth changing this, I mainly wanted to clarify the comment.